src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,39 @@
 
 public class SalaryInfo {
```

Comment:
It's better to use DateTimeFormatter and LocalDate. 
Also, you can create separate method isInRangeOfPeriod 
somethind like 
```
for (String name : names) {
            int totalSalary = 0;
            for (String values : data) {
                String[] dataInfo = values.split(" ");
                if (isInRangeOfPeriod(dataInfo[0], dateFrom, dateTo)
                        && name.equals(dataInfo[1])) {
```